github-actions/run-github-release/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Run GitHub Release
+description: Fetches and runs any GitHub released binary.
+
+inputs:
+  args:
+    description: Arguments to pass to the binary. Default: no arguments
+    required: false
+  bin:
+    description: The binary file stem e.g. `mdbook`. Default: Same as repo name.
+    required: false
+  owner:
+    description: The owner of the GitHub repository e.g. `rust-lang`
+    required: false
+    default: rust-lang
+  regex:
+    description: The regex to match against the name of the asset release.
+    required: true
+  repo:
+    description: The name of the repository containing the releases.
+    required: true
+  token:
+    description: GitHub token for authenication.
+    required: true
+
+runs:
+  using: node12
```

Comment:
What is stopping you from migrating to a higher node version? This version will show a deprecation warning, and [GitHub will soon drop support](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/).